bin/si-sdf-api-test/tests/1-create_and_apply_across_change_sets.ts

```diff
@@ -1,75 +1,96 @@
 import assert from "node:assert";
 import { SdfApiClient } from "../sdf_api_client.ts";
-import { extractSchemaVariant, getSchemaVariants } from "../test_helpers.ts";
-import abandon_all_change_sets from "../benchmark/abandon_all_change_sets.ts";
+import { abandon_all_change_sets, createComponent, createComponentPayload, eventualMVAssert, getVariants, getViews, installModule } from "../test_helpers.ts";
 
 export default async function create_and_and_apply_across_change_sets(
   sdfApiClient: SdfApiClient,
 ) {
   return await create_and_and_apply_across_change_sets_inner(sdfApiClient);
 }
-
+const TEST_RESOURCE_ACTION_MODULE_ID = "01JMJB1QW3M8D7MNN8NGBR739J";
```

Comment:
I like this because if this slips out from under us, it should be easy to catch.